File: Selecao I/abs.cpp
/*@abs: Valor absoluto*/

#include <iostream>
#include <locale>
using namespace std;

int main(){
    setlocale(LC_ALL, "Portuguese");
    
    int n1, n2;

    cin >> n1;
    cin >> n2;

    int diferenca = (n1 - n2);
    
    if(diferenca < 0){
        diferenca = diferenca * -1;
        cout << diferenca << endl;
    }else{
        cout << diferenca << endl;
    }
    
    return 0;
}

File: src/Selecao_II/Abs.java
/**
 * Questão: @abs: Valor absoluto
 *
 * @author UxieGu1
 */
package Selecao_II;
import java.util.Scanner;

public class Abs {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int n1 = scan.nextInt();
        int n2 = scan.nextInt();

        int diferenca = Math.abs(n1 - n2);
        System.out.println(diferenca);

        scan.close();
    }
}
